docs/source/virtual_ecosystem/implementation/abiotic_implementation.md

```diff
@@ -45,6 +45,45 @@ display_markdown(
 )
 ```
 
+### A note on pre-processing
+
+The atmospheric variables from regional climate models, observations, or reanalysis are
+typically provided in spatial resolutions that are much coarser than the
+requirements of the Virtual Ecosystem, and follow different naming and unit conventions.
```

Comment:
I'd probably have this as a separate page or use it as an include rather than duplicate it across the two models